src/components/create/create.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../utils/helpers/tags/tags';
+import CreateStyle from './create.style';
+
+class Create extends React.Component {
+  constructor(args) {
+    super(args);
+    this.linkProps = this.linkProps.bind(this);
+  }
+
+  /** Returns the props for the component. */
+  linkProps() {
```

Comment:
If you turn this into an arrow function, you can delete the class `constructor`.